refcount.go

```diff
@@ -58,7 +57,7 @@ func (c *RefCount) Retain() {
 // If an Action is configured, then this action will be run once the
 // refcount becomes free.
 func (c *RefCount) Release() bool {
-	switch c.count.Dec() {
+	switch c.count.Add(^uint32(0)) {
```

Comment:
I'm a bit confused on why you're adding `^uint32(0)` instead of `uint32(-1)` like in the original implementation.

Reply:
Thanks for clarifying!